controllers/drpolicy_controller.go

```diff
@@ -243,6 +250,36 @@ func s3ProfileValidate(ctx context.Context, apiReader client.Reader,
 	return "", nil
 }
 
+func validateCIDRsFormat(drpolicy *ramen.DRPolicy, log logr.Logger) error {
+	// validate the CIDRs format
+	for i := range drpolicy.Spec.DRClusterSet {
+		cluster := &drpolicy.Spec.DRClusterSet[i]
+		if err := CIDRsFomatValidate(cluster.CIDRs, log); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func CIDRsFomatValidate(cidrs []string, log logr.Logger) error {
```

Comment:
no need to export this function, can the function be better named? `validateCIDRsFormat` and `CIDRsFomatValidate` functions names seems same.

Reply:
you can add unit tests?

Reply:
yes, the functions seem similar. :). I will try to add the unit tests too.

Reply:
functions renamed